docs/configs-api.md

```diff
@@ -0,0 +1,95 @@
+# Configs API
+
+The configs api provides API-driven multi-tenant approach to prometheus' Rule files and Alertmanager configs.
+
+Each tenant has a single Rule file and Alertmanager Configuration. A POST operation will effectively replace the existing copy with the configs provided in the request body.
+
+Although Prometheus works with YAML files by default, these HTTP API's only accept JSON. You may convert your YAML to JSON using a tool such as [yaml2json](https://github.com/bronze1man/yaml2json).
```

Comment:
I remember succeeding in posting a raw YAML file at one point... that said I think using JSON makes the requests much simpler to figure out.

Reply:
I'm certain we post yaml files in our configuration system to this api. @rsteneteg , can you provide some comments here?

Reply:
The part about only accepting JSON is correct, however just to be clear it's only the request to config service that needs to be in JSON format, the contents of your rule files should still be YAML if using the prometheus v2 rules format